assessments/projects/contour/self-assessment.md

```diff
@@ -0,0 +1,208 @@
+# Contour Self-assessment
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/projectcontour/contour |
+| Security Provider | No |
+| Languages | Go |
+| SBOM | TBD after discussion with project maintainers |
```

Comment:
```suggestion
| SBOM | Contour does not currently produce an SBOM at build time |
```

The target state for all of the projects is to have an automatic generation of SBOMs during the build, to be included with the release. Some other projects have generated SBOMs as part of this activity, but that's not necessary or valuable for the purpose of the self-assessment.